library/src/main/java/com/novoda/downloadmanager/lib/DownloadService.java

```diff
@@ -139,9 +139,14 @@ public void onCreate() {
         ContentResolver contentResolver = getContentResolver();
         File downloadDataDir = StorageManager.getDownloadDataDirectory(this);
         File externalStorageDir = Environment.getExternalStorageDirectory();
+        File[] externalStorageDirs = new File[0];
+        if (android.os.Build.VERSION.SDK_INT >= 19 )
```

Comment:
also @Hackmodford it'd be awesome if you can use the novoda IDE settings for the formatting when you submit a PR in order to keep some consistency in the codebase 😉  https://github.com/novoda/novoda/tree/master/ide-settings/intellij